reedstrm/baking_workflow_and_errors.md

```diff
@@ -0,0 +1,120 @@
+# Baking and Recipes
+
+Baking has become a critical step for publishing - however, it is not planned for it to cause publishing to fail. This is partly due to how long baking may take - much like PDF generation, it can take several hours, if there are external parts, like exercises, that need to be fetched and incorporated into the pages.
+
+This slightly odd situation (is baking a publish blocker, or not?) comes about partly due to derived copies - publishing a new version of content impacts not only the immediate book, but any book that shares pages with it. It was decided that failure of related books to bake is _not_ a publication blocker, so by extension, it is not for any book. However, we want to be careful to not break existing books by publishing new content, but we cannot test every book that contains a given page.
+
+## Selection of Recipe and Fallback
+
+So, we will implement two ways to mitigate this potential problem. First, will we keep track of when and with what recipe a given book successfully baked. This will point to the actual recipe bits - the file used. In order to use a given recipe for more than one book, we are extending the `print_style` attribute to also select a default recipe. Note that this can be updated independently of re-baking a given book.
```

Comment:
oh, in that case I was suggesting > 1 field:

- the actual CSS code
- a label ("ccap-biology")
- a field that supports adding a tag name or a sha so when baking breaks folks know which version of that CSS code they should look at
- optionally a timestamp of when the CSS code was uploaded